app/views/playlists/_playlist_white.html.erb

```diff
@@ -49,9 +49,11 @@
     <ul class="external-links">
       <% [:link1, :link2, :link3].collect do |link| %>
         <% if valid_link = @playlist.send(link) %>
+          <% if external_link_for(valid_link) %>
```

Comment:
Thanks for this, it will work for now. It will have to be refactored eventually – `external_link_for` is checking to see if the link exists, but what we want is to have that check happen before the `<li>` is written. That means we should either put the check for presence outside the `li` or put the `li` inside the helper (these days i don't like hiding html in helpers, really...)